plugins/direct-csi.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: direct-csi
+spec:
+  version: v1.0.0-rc-3
```

Comment:
Please only release "final" versions. This index has a bot that automatically approves version bumps. But it will not work with release candidates.